src/Sylius/Bundle/CoreBundle/Validator/Constraints/HasEnabledEntityValidator.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Validator\Constraints;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+/*
+ * @author Gustavo Perdomo <gperdomor@gmail.com>
+ */
+class HasEnabledEntityValidator extends ConstraintValidator
```

Comment:
Sounds logic, but i prefer the <code>entity</code> term, because when talk about things which must be persisted in database, is more natural use the <code>entity</code> term, even when the entity indeed, is an object, at least for me